climada_petals/entity/exposures/black_marble.py

```diff
@@ -102,22 +105,23 @@ def set_countries(self, countries, ref_year=2016, res_km=None, from_hr=None,
 
         bkmrbl_list = []
 
-        description = ""
+        descr_lines = []
         for cntry_iso, cntry_val in cntry_info.items():
 
             bkmrbl_list.append(
                 self._set_one_country(cntry_val, nightlight, coord_nl, res_fact, res_km,
                                       cntry_admin1[cntry_iso], **kwargs).gdf)
-            description += ("{} {:d} GDP: {:.3e} income group: {:d} \n").\
-                format(cntry_val[1], cntry_val[3], cntry_val[4], cntry_val[5])
+            descr_lines.append(f"{cntry_val[1]} {cntry_val[3]:d}"
+                            f" GDP: {cntry_val[4]:.3e} income group: {cntry_val[5]:d}")
 
-        Exposures.__init__(
+        BlackMarble.__init__(
             self,
             data=Exposures.concat(bkmrbl_list).gdf,
             crs=DEF_CRS,
             ref_year=ref_year,
-            tag=Tag(file_name=fn_nl, description=description),
-            value_unit='USD'
+            value_unit='USD',
+            description="\n".join(descr_lines),
```

Comment:
How does this work? The `description` is not added to the `Exposures._metadata`, so I assume it is *not* added as attribute but passed on to the `GeoDataFrame` constructor? How does that one deal with a string as argument? I am quite certain something goes wrong here.

Reply:
Good question! I cheated there and sneaked `description` in as an attribute of the base class: https://github.com/CLIMADA-project/climada_python/commit/438be81def2488da58319823bf8a8b11d5a21616
Kind of had to otherwise the two repositories would have been in a catch 22 state. 😏

Reply:
I see 👍